packages/dev-server/package.json

```diff
@@ -70,5 +70,9 @@
     "typedoc": "^0.25.3",
     "typedoc-plugin-markdown": "^3.17.1",
     "typescript": "^5.2.2"
+  },
+  "peerDependencies": {
+    "@react-native-community/cli-debugger-ui": ">=11.3.1",
+    "@react-native-community/cli-server-api": ">=11.3.1"
```

Comment:
I'd change this to `peerDependenciesMeta`, otherwise npm will install these deps (which are transitive deps of `react-native` currently and don't normally show up in app's dependencies)